package.json

```diff
@@ -1,19 +1,43 @@
 {
-  "name": "scratch-editor-translations",
-  "version": "0.5.0",
-  "description": "Translation file for the Scratch 3.0 Editor",
-  "main": "index.js",
+  "name": "scratch-l10n",
+  "version": "0.1.0",
+  "description": "Localization for the Scratch 3.0 components",
+  "main": "./dist/l10n.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build:data": "babel-node scripts/build-data",
+    "build": "npm run clean && npm run build:data && webpack --progress --colors --bail",
+    "clean": "rimraf ./dist ./locales && mkdirp dist locales",
+    "lint": "eslint . && jlint --silent --glob './gui/*.json'",
```

Comment:
In www we do json linting via eslint and a plugin — I think that would be simpler than a separate linter for json.

Reply:
Definitely!